Source/Voidsinger/Voidgrids/Parts/PartModules/ActivatablePartModules/ResourceCreationModule.cpp

```diff
@@ -0,0 +1,41 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "ResourceCreationModule.h"
+#include "Voidsinger/Voidgrids/Parts/Part.h"
+#include "Voidsinger/Voidgrids/Voidgrid.h"
+
+/* \/ ======================= \/ *\
+|  \/ UResourceCreationModule \/  |
+\* \/ ======================= \/ */
+
+/* ---------------- *\
+\* \/ Activation \/ */
+
+/**
+ * Override of OnActivate to create resources
+ *
+ * @param Data - The data of this activation
```

Comment:
Not helpful, What is the data of this activation, and how does it effect the amount of resources created